lib/ansible/modules/packaging/os/yum_repository.py

```diff
@@ -391,6 +385,8 @@
   - The repo file will be automatically deleted if it contains no repository.
   - When removing a repository, beware that the metadata cache may still remain
     on disk until you run C(yum clean all). Use a notification handler for this.
+  - "The C(params) parameter was removed in Ansible-2.5 due to circumventing Ansible's parameter
```

Comment:
Maybe just say 'Ansible 2.5' rather than having the dash.

Reply:
Fixed.